database.go

```diff
@@ -0,0 +1,78 @@
+package authenticatedblockstore
+
+import (
+	"fmt"
+	"log"
+)
+
+type DataBase struct {
+	tableManager *TableManager
+	logManager   *LogManager
+}
+
+// Exported Package APIs.
+// These are called by: package.api()
+//
+//
+func Create() (*DataBase, error) {
+	fmt.Println("Creating a new DataBase, returning a handle to the caller.")
+	var newDB *DataBase
+
+	tb, err := NewTableManager()
+	if err != nil {
+		log.Fatal(err)
+	}
+	newDB.tableManager = tb
+
+	lm, err := NewLogManager()
+	if err != nil {
+		log.Fatal(err)
+	}
+	newDB.logManager = lm
+	return newDB, nil
+}
+
+func Open() (*DataBase, error) {
+	fmt.Println("Opening an existing DataBase, returning a handle to the caller.")
+	var newDB *DataBase
+	return newDB, nil
```

Comment:
I think you can just write this as 

return &DataBase{}, nil

Reply:
As of now this a placeholder, I thought it probably should have more stuff in it.